### DISCUSSION TITLE
A setting for changing Clip Skip from the default?

### DISCUSSION THREAD

**Discussion by Lolcupcakes** [jordenyt/stable_diffusion_sketch#18] [Ideas]
As different models work better with different Clip Skip settings and some even recommend changing it. If that seems like a fair suggestion, that is. Thank you for considering ^^

**Comment by jordenyt** [comment]
Add support for Clip Skip in version 0.15.2.  You may set the default value in the SD Settings, or set in the mode JSON for specific mode.

**Reply by Lolcupcakes** [reply]
Thank you so much! That'll help a ton.
